src/main/scala/mesosphere/marathon/health/HealthCheckWorkerActor.scala

```diff
@@ -73,6 +73,13 @@ class HealthCheckWorkerActor extends Actor with ActorLogging {
       }
     }
 
+  def getPort(task: MarathonTask, check: HealthCheck): Option[Integer] = {
```

Comment:
Yeah, this complexity checker is not very smart. It should treat inline defs as separate methods.